nx-X11/include/extensions/Imakefile

```diff
@@ -42,15 +42,6 @@ XINERAMAHEADERS = Xinerama.h panoramiXext.h panoramiXproto.h
 #if BuildRender || BuildXftLibrary
 RENDERHEADERS = render.h renderproto.h
 #endif
-#if BuildXfixes || BuildXfixesLibrary
-XFIXESHEADERS = xfixeswire.h xfixesproto.h
-#endif
-#if BuildDamage || BuildDamageLibrary
-DAMAGEHEADERS = damagewire.h damageproto.h
-#endif
-#if BuildComposite || BuildCompositeLibrary
-COMPOSITEHEADERS = composite.h compositeproto.h
-#endif
```

Comment:
Removing these lines is potentially  problematic, because `BuildXfixes`, `BuildDamage` and `BuildComposite` are set to `YES` in `nx-X11/config/cf/X11.tmpl`.

If `BuildXfixes`, `BuildDamage` and `BuildComposite` mean, that the internal header files should be used, it’s probably better to either unset or set thee to `NO` (not sure what method is correct for imake) in `nx-X11/config/cf/X11.tmpl`.